source/common/quic/platform/quiche_mem_slice_impl.h

```diff
@@ -14,41 +14,41 @@
 
 #include "quiche/quic/core/quic_buffer_allocator.h"
 
-namespace quic {
+namespace quiche {
 
 // Implements the interface required by
 // https://quiche.googlesource.com/quiche/+/refs/heads/master/quic/platform/api/quic_mem_slice.h
```

Comment:
We probably need to update this comment as well.

Reply:
good point!